go.mod

```diff
@@ -36,13 +36,13 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/prometheus/client_model v0.4.0
 	github.com/prometheus/common v0.43.0
-	github.com/rancher/lasso v0.0.0-20230502204209-3eb356f3e8cf
+	github.com/rancher/lasso v0.0.0-20230830164424-d684fdeb6f29
 	github.com/rancher/rancher/pkg/apis v0.0.0
 	github.com/rancher/wrangler v0.8.11
 	github.com/sirupsen/logrus v1.9.0
-	github.com/stretchr/testify v1.8.2
-	k8s.io/api v0.26.4
-	k8s.io/apimachinery v0.26.4
+	github.com/stretchr/testify v1.8.4
+	k8s.io/api v0.27.5
+	k8s.io/apimachinery v0.27.5
 	k8s.io/client-go v12.0.0+incompatible
```

Comment:
What's the reason for using this `v12...` version of client-go instead of [v0.27.5](https://pkg.go.dev/k8s.io/client-go@v0.27.5)?

Reply:
This is overriden by the replace directive above: https://github.com/rancher/csp-adapter/pull/65/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R13. Rancher/rancher uses a similar system: https://github.com/rancher/rancher/blob/564fbe67d735feab5064f8bc4cbd42df730fe550/go.mod#L152.

I'd appreciate it if you could focus on the changes which this PR made - given the status of the repo and the urgency of this I think it makes sense to focus on the new-side changes rather than re-evaluating prior decisions.

Reply:
I think it doesn't matter because this gets replaced in the `replace` directive:

```
replace (
	...
	k8s.io/client-go => github.com/rancher/client-go v1.27.4-rancher1
```